From 59c3ff750831338d05ab67d5efd7fc101c451aff Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@windriver.com>
Date: Thu, 13 Sep 2012 16:46:45 -0400
Subject: [PATCH] trace/Kconfig: add PERF_EVENTS to UPROBE_EVENT selected
 options

PERF_EVENTS must be built to provide the functionality that
UPROBE_EVENT requires, or we end up with linkage errors.

Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 kernel/trace/Kconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/kernel/trace/Kconfig b/kernel/trace/Kconfig
index ea4bff6..8511b79 100644
--- a/kernel/trace/Kconfig
+++ b/kernel/trace/Kconfig
@@ -392,6 +392,7 @@ config UPROBE_EVENT
 	depends on MMU
 	select UPROBES
 	select PROBE_EVENTS
+	select PERF_EVENTS
 	select TRACING
 	default n
 	help
-- 
1.7.5.4

